src/components/Feed/Feed.js

```diff
@@ -44,14 +44,15 @@ const Feed = ({ profileIsVisible }) => {
       return postsToDisplay.map(post => {
         return (
           <Post
-            userName={post.userName}
-            title={post.title}
-            date={post.date}
-            location={post.location}
-            description={post.description}
-            distance={post.distance}
-            imageUrl={post.image}
-          />
+          userName={post.userName}
+          title={post.title}
+          date={post.date}
+          location={post.location}
+          description={post.description}
+          distance={post.distance}
+          imageUrl={post.image}
+          key={post.id}
```

Comment:
Nice catch! I believe we were getting an error for not having a key.

Reply:
_Yes, and..._ my intention here _was_ to resolve exactly that error, and this appears to be the source the error points to (I think this is the only place we're creating through iteration where such a thing would even be an issue), however I am still getting that error in my console even with this change. I have learned before that the key needs to be on the parent element if it were wrapped, but still looking for other ideas to fully resolve this one.

![E75EA609-30E7-4D06-AFB9-2A0D0162F75F_4_5005_c](https://user-images.githubusercontent.com/64617223/115431755-e30dec80-a1c2-11eb-81ac-b3747043b7c5.jpeg)

Reply:
My guess is the data right now doesn't have an 'id' property, so would that be reading anything right now? It seems like it should work when we use data with an 'id' property. We do have a 'userID' on the mock data, but there are two posts with the same 'userID' so that wouldn't be unique in that case.